PrivateHeaders/APPX/ZIP.h

```diff
@@ -234,10 +234,11 @@ namespace appx {
         // names, hence the use of sanitizedFileName
         static const std::unordered_map<std::string, const char *>
             sKnownContentTypes = {
+                {"appx", "application/vnd.ms-appx"},
```

Comment:
We are skipping appx files from AppxBlockMap.xml but not from [Content_Types].xml